### PR TITLE
#902 Fix iterator test of CollectionOfTest

### PR DESCRIPTION
**Pull request by DanielPalafox**
I replaced iterator with iterable for #902 as it is discussed in #911

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#938](https://github.com/yegor256/cactoos/issues/938) is now in scope, role is `REV`

**Comment by svendiedrichsen** [comment]
@DanielPalafox Shouldn't the ctor using the `Iterator` be removed? And thus all usages of this ctor be replaced with the usage of the ctor using an `Iterable`?

**Comment by DanielPalafox** [comment]
@svendiedrichsen this specific task was about resolved a test who wasn't working properly with an iterator, remove all the usages and remove the ctor it should be done in #911. I updated my comment referencing the task to avoid misunderstandings.

**Comment by svendiedrichsen** [comment]
@DanielPalafox Please see @llorllale first comment to #902

File: src/test/java/org/cactoos/collection/CollectionOfTest.java [hunk 1]
```diff
@@ -51,12 +46,11 @@ public void behavesAsCollection() throws Exception {
         );
     }
 
-    @Ignore
     @Test
-    public void buildsCollectionFromIterator() throws Exception {
+    public void buildsCollectionFromIterable() throws Exception {
         MatcherAssert.assertThat(
             "Can't build a collection from iterator",
```
Comment (llorllale) [inline src/test/java/org/cactoos/collection/CollectionOfTest.java]: @DanielPalafox should say "iterable" now

**Comment by llorllale** [comment]
@DanielPalafox I left you one minor comment, see above

**Comment by DanielPalafox** [comment]
@llorllale Thanks for the review, correction applied, please take a look

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#938` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#938](https://github.com/yegor256/cactoos/issues/938) is now out of scope

**Pull request merged**